### DISCUSSION TITLE
Double goto definition reference

### DISCUSSION THREAD

**Discussion by Lun4m** [latex-lsp/texlab#922] [Q&A]
I just installed TexLab with [Mason](https://github.com/williamboman/mason.nvim) in [LazyVim](https://github.com/LazyVim/LazyVim) (nvim 0.9.1). 
When I `gd` to open a file included with `\include{}` I get two references pointing at the same file.
As far as I can tell, I only have a single instance of TexLab running, and the problem doesn't appear when I `gd` references from `\ref{}`.
Anyone has any idea on what could be causing this issue?

![Screenshot from 2023-08-22 17-45-28](https://github.com/latex-lsp/texlab/assets/15078179/dbe908e8-3a0d-4ce2-b544-367022ecd3b9)

**Comment by xyecoding** [comment]
I have a similar problem. Dose the problem be solved?
